fcrepo-http-commons/src/main/java/org/fcrepo/http/commons/exceptionhandlers/InvalidChecksumExceptionMapper.java

```diff
@@ -44,7 +44,7 @@
     @Override
     public Response toResponse(final InvalidChecksumException e) {
 
-        LOGGER.info(e.toString());
+        LOGGER.error(e.toString());
```

Comment:
I realize you did not create this issue, but could you please make this log statement an actual statement with a message, like the others in this ticket?